hindsight/Resources/LoginButton.swift

```diff
@@ -0,0 +1,138 @@
+//
+//  LoginButton.swift
```

Comment:
I don't think this file belongs to `Resources`, any idea?

Reply:
Agree, we should created a separate component folder to put all the reusable UI components.

Reply:
I have created a Components folder and moved the LoginButton.swift to the folder.